driver/src/main/java/com/code_intelligence/jazzer/junit/FuzzTest.java

```diff
@@ -50,17 +50,6 @@
 // from parallelization.
 @ResourceLock(value = "jazzer", mode = ResourceAccessMode.READ_WRITE)
 public @interface FuzzTest {
```

Comment:
JavaDoc still mentions `seedCorpus`.

Reply:
Removed that part, the docs will have to be rewritten anyway.